contracts/NFTStorage.sol

```diff
@@ -586,7 +586,7 @@ contract NFTStorage is IERC721ReceiverUpgradeable, Initializable, AccessControlU
 
         mintedId =
             characters.customMint(address(this), xp,
-            level, trait, seed, mintedId, bonusPower, reputation);
+            level, trait, seed, mintedId, bonusPower, reputation, 0);
```

Comment:
NFTStorage should always mint "version" 0?

Reply:
This is the old bridge. NFTStorage only holds NFTs bridged a long time ago, only gen1 could have gone in there.